com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/RestUtils.java

```diff
@@ -272,4 +274,15 @@ public static String getKeyColumnName(String tableName) {
 
 		return keyColumns[0];
 	}
+	
+	/**
+	 * Defines whether the UUID value of a lookup record is returned in the response  
+	 * @param tableName
+	 * @return true if the lookup table should return the UUID in the REST response
+	 */
+	public static boolean isReturnUULookup(String tableName) {
+		String exportedUUTables = MSysConfig.getValue(EXPORT_UU_LOOKUP_SYSCONFIG_NAME, Env.getAD_Client_ID(Env.getCtx()));
+		return !Util.isEmpty(exportedUUTables) && 
+				(exportedUUTables.equals("ALL") || exportedUUTables.toLowerCase().contains(tableName.toLowerCase()));
```

Comment:
this looks wrong here.  Comparing with contains is dangerous because for example ad_field will match ad_field and also ad_fieldgroup.

I think the REST_TABLES_EXPORT_LOOKUP_UU must be a comma separated list of tablenames, and you better convert it here to an array (String.split) and then check if the tablename is within the array

In the documentation would be good to add an example of how to fill REST_TABLES_EXPORT_LOOKUP_UU ?

Reply:
Fixed in new commit. I will update the documentation when the PR is approved.